Documentation/common-issues.md

```diff
@@ -518,3 +519,23 @@ The only solution to this problem is to upgrade your kernel to `4.7` or higher.
 This is due to a mount flag added in the kernel version `4.7` which allows to chose the filesystem by name.
 
 For additional info on the kernel version requirement for multiple shared filesystems (CephFS), see [Filesystem - Kernel version requirement](ceph-filesystem.md#kernel-version-requirement).
+
+# Activate log to file for a particular Ceph daemon
+
+They are case where looking at Kubernetes logs is not enough for diverse reasons, but just to name a few:
+
+* not everyone is familiar for Kubernetes logging and expects to find logs in traditional directories
+* logs got eaten (buffer limit from the log engine) and thus not requestable from Kubernetes
+
+So for each daemon, `dataDirHostPath` is used to store logs, if logging is activated.
+Rook will bindmount `dataDirHostPath` for every pod.
+As of Ceph Nautilus 14.2.1, it is possible to enable logging for a particular daemon on the fly.
+Let's say you want to enable logging for `mon.a`, but only for this daemon.
+Using the toolbox or from inside the operator run:
+
+```
+ceph config daemon mon.a log_to_file true
+```
+
+This will activate looging on the filesystem, you will be able to find logs in `dataDirHostPath/$NAMESPACE/log`, so typically this would mean `/var/lib/rook/rook-ceph/log`.
+You don't need to restart the pod, the effect will be immediate.
```

Comment:
Do we need an instruction to disable the logging when they're done? Otherwise they might have to worry about  large logs over time.